src/renderer/components/ui/SimpleColorPicker.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+
+export default class SimpleColorPicker extends React.Component {
+  readonly props: {
+    label: string,
+    value: string,
+    isVisible: boolean,
+    isEnabled: boolean,
+    children?: React.ReactNode,
+    onChange: (value: string) => void
+  };
+
+    render() {
+        if (this.props.isVisible) {
```

Comment:
Please don't put `isVisible` in this component. Use `this.props.scene.backgroundType== BT.color && <SimpleColorPicker...` instead where the component is used.

Reply:
Ok give me an hour or so